Signal/src/ViewControllers/ConversationView/Cells/AudioMessageView.swift

```diff
@@ -426,7 +426,7 @@ class AudioMessageView: ManualStackView {
         }
 
         let locationInSlider = convert(point, to: waveformProgress)
-        return locationInSlider.x >= 0 && locationInSlider.x <= waveformProgress.width
+        return waveformProgress.bounds.contains(locationInSlider)
```

Comment:
There is another bug here that I thought about looking at but decided against it since it might require design input. Currently, the location of the scrubber isn't consulted here. This means that if the progress is at 0% but you start panning at 90% the scrubber will immediately jump to 90%. Kinda weird.

Here's what I think might make sense but feel free to ignore. Just my $0.02:
- If you are physically touching the screen to drag, you should start the pan reasonably close to the scrubber. Ignore anything that's outside of some target area.
- If you're panning with the trackpad, you can start a pan anywhere in the waveform but the scrubber will move linearly with the pan distance. It shouldn't immediately jump to the cursor location.